spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -87,7 +87,7 @@ main() {
     cd $CURRENT_DIR
 
     #TODO what happens if cluster is already there????
-    "${KIND}" create cluster --config=kind-config.yaml --loglevel=debug
```

Comment:
`--loglevel=debug` is deprecated; instead `-v 2147483647`; its like saying "maximum logging output"